typed/clj.analyzer/src/typed/clj/analyzer/utils.cljc

```diff
@@ -500,10 +500,10 @@
    subset of methods that match best the given tags"
   [tags methods]
   (let [o-tags (mapv #(or (maybe-class %) Object) tags)]
-    (if-let [methods' (or (not-empty
+    (if-let [methods (or (not-empty
```

Comment:
I've kept this because I didn't quite like the fact that `if-let` doesn't shadow if the value evaluates to falsey. Kinda confusing that `methods` refers to different things in `then` and `else` branches of the `if-let`.

Reply:
I'm more worried about accidentally using the old value.